### PR TITLE
TiclFR report

### PR DESCRIPTION
**Pull request by leondavis1**
Introduces a report for TICL_FR

### CODE REVIEW

File: ramutils/events.py [hunk 1]
```diff
@@ -1221,6 +1225,55 @@ def get_stim_list_mask(events):
...
+
+    """
+    if 'list_phase' in events.dtype.names:
+        return events
+
+    dtype_desc = events.type.dtype.str
+
+    phases = np.empty(len(events), dtype=dtype_desc)
+
+    lstphases = ['']
```
Comment (mivade) [inline ramutils/events.py]: Why doesn't this start with an empty list?

File: ramutils/powers.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 import pandas as pd
 from ptsa.data.filters import (
     MonopolarToBipolarMapper,
-    MorletWaveletFilterCpp
+    MorletWaveletFilter
```
Comment (mivade) [inline ramutils/powers.py]: Does this mean this PR also upgrades to PTSA 2.0 and closes #237?

Comment (leondavis1) [reply ramutils/powers.py]: It does now

File: ramutils/events.py (hunk 1 above)
Comment (leondavis1) [reply ramutils/events.py]: ... No good reason. Fixed now.

**Pull request merged**